add-ons/dspace-docker-tools/ingestAIP.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+# AIPZIP=${AIPZIP:-https://github.com/DSpace-Labs/AIP-Files/raw/master/DogPhotosAIP.zip}
+CHECKFILE=/dspace/assetstore/ingest.hasrun.flag
+
+export JAVA_OPTS="${JAVA_OPTS} -Xmx2500m -Dupload.temp.dir=/dspace/upload -Djava.io.tmpdir=/tmp"
+
+if [ ! -f $CHECKFILE ]
+then
+  /dspace-docker-tools/createAdmin.sh
+  AIPZIP=${AIPZIP:-https://github.com/DSpace-Labs/AIP-Files/raw/master/dogAndReport.zip}
+  ADMIN_EMAIL=${ADMIN_EMAIL:-test@test.edu}
+  if [ ! -z $AIPZIP ]
+  then
+    AIPDIR=/tmp/aip-dir
+    rm -rf ${AIPDIR}
```

Comment:
I‘d considder this a bit dangerous. Maybe add some output with echo and `rm -i` to get approval by the user?

Reply:
This action happens in the background as the container is started, so there is no opportunity for user input.  This is running in /tmp, so it should only impact temporary files.